stage0/00-configure-apt/files/sources.list

```diff
@@ -1,3 +1,4 @@
 deb http://raspbian.raspberrypi.org/raspbian/ buster main contrib non-free rpi
+deb http://pyxis-repo.s3.eu-north-1.amazonaws.com/pyxis-0.1/ pyxian main
```

Comment:
We should make the URL more generic, not specific to "pyxis". Also, alias it under renetec.io.

Reply:
Who can make alias to pyxis-repo.s3.eu-north-1.amazonaws.com?

Reply:
I will do. Let's merge it this way for now and update later.